get-started/start-building/batch.md

```diff
@@ -244,29 +311,42 @@ Use the following Python script or curl command examples to retrieve results fro
         -H "Authorization: Bearer $API_KEY" > batch_output.jsonl
     ```
 
+??? code "View the complete script"
+
+    === "Python"
+
+        ```python
+        --8<-- 'code/get-started/start-building/batch/batch-jsonl-01.py'
+        ```
+
 ## List all batch jobs
 
 To list all of your batch jobs, send a request to the `batches` endpoint without specifying a `batch_id`. To constrain the query response, you can also use a `limit` parameter.
 
-Use the following Python script or curl command examples to list all of your batch jobs:
+You can use the following snippets to list all of your batch jobs:
 
 === "Python"
 
     ```python
     from openai import OpenAI
-
-    # Configure OpenAI client
+    from getpass import getpass
+    
+    # Get API key from user input
+    api_key = getpass("Enter your kluster.ai API key: ")
```

Comment:
Should probably use the usual syntax here ("INSERT_API_KEY") and move this to a code snippet